packages/atlas/src/components/_crt/MarketDrawer/MarketDrawer.tsx

```diff
@@ -0,0 +1,251 @@
+        }
+      case MARKET_STEPS.saleSummary:
+        return {
+          text: 'Back',
+          onClick: () => {
+            flushSync(() => {
+              setIsGoingBack(true)
+            })
+            setActiveStep(MARKET_STEPS.market)
+          },
+        }
+    }
+  }, [activeStep, isDirty, onClose, openDialog])
+
+  const primaryButton = useMemo(() => {
+    switch (activeStep) {
+      case MARKET_STEPS.market:
+        return {
+          text: 'Next',
+          onClick: () => {
+            setActiveStep(MARKET_STEPS.saleSummary)
+          },
+        }
+      case MARKET_STEPS.saleSummary:
+        return {
+          text: 'Start sale',
+          onClick: () => {
+            //token sale tx
+          },
+        }
+    }
+  }, [activeStep])
+
+  const stepContent = () => {
+    switch (activeStep) {
+      case MARKET_STEPS.market:
+        return (
+          <div>
+            <Text variant="h500" as="span" margin={{ bottom: 2, right: 4 }}>
+              Market
+            </Text>
+            <TextButton as="span" icon={<SvgActionPlay />} iconPlacement="left" color="colorTextPrimary">
+              Learn more
+            </TextButton>
+            <Text variant="t300" color="colorText" as="p">
+              Automated market maker (AMM) will increase ${tokenName} price after each purchase and decrease its price
+              when someone sells it to the AMM.
+            </Text>
+            <Text variant="h300" as="h1" margin={{ top: 8, bottom: 2 }}>
+              Starting price for token
+            </Text>
+            <Text variant="t100" color="colorText" as="div" margin={{ bottom: 4 }}>
+              You cannot set price lower than <NumberFormat value={DEFAULT_MIN_PRICE} as="span" withToken />
+            </Text>
+            <Controller
+              control={control}
+              render={({ field: { value: price, onChange: setPrice } }) => (
+                <TokenInput
+                  value={price}
+                  onChange={setPrice}
+                  nodeEnd={
+                    <Text variant="t300" as="p" color="colorTextMuted">
+                      ${tokenInUsd.toFixed(2)}
+                    </Text>
+                  }
+                />
+              )}
+              name="price"
+            />
+            <Text variant="h300" as="h1" margin={{ top: 8, bottom: 2 }}>
+              Terms and conditions
+            </Text>
+            <Text variant="t100" color="colorText" as="p" margin={{ bottom: 4 }}>
+              Change default rules if you want to add some additional terms.
+            </Text>
+            <Controller
+              control={control}
+              render={({ field: { value: isChecked, onChange } }) => (
+                <Checkbox
+                  value={isChecked}
+                  label="Keep the default terms & conditions"
+                  onChange={(checked) => {
+                    if (checked) {
+                      resetField('tnc')
```

Comment:
I think it won't bring back the default tnc, but can't check on story

Reply:
Why not? added this call programmatically, field value was reset